ledfx/effects/utils/logsec.py

```
@@ -0,0 +1,79 @@
+            self.lasttime = nowint
+        self.log = result
+
+    def try_log(self):
+        end = timeit.default_timer()
+        r_time = end - self.current_time
+        self.r_total += r_time
+        if self.log is True:
+            if self.fps > 0:
+                r_avg = self.r_total / self.fps
+            else:
+                r_avg = 0.0
```

Ruff

Use ternary operator `r_avg = self.r_total / self.fps if self.fps > 0 else 0.0` instead of `if`-`else`-block

Replace `if`-`else`-block with `r_avg = self.r_total / self.fps if self.fps > 0 else 0.0`

(SIM108)